lib/pipeline.js

```diff
@@ -135,6 +135,82 @@ Pipeline.prototype = {
   },
 
   /**
+   * Bind to a streams events, and re-emit them on the pipeline
+   * @param {Stream} stream
+   * @param {String|Array} events
+   * @returns {Pipeline}
+   * @chainable
+   */
+  bind( stream, events ) {
```

Comment:
Thanks for taking the time to try and explain things to a streams newbie ;-)

So, given all the above should there be unit tests for binding an event on a stream that actually isn't in the pipeline yet? (what's the expected behaviour in that case?)

Also, you still need to rebuild `doc/README.md` to pick up the doc-string fixes you made.